onnxruntime/core/graph/function_utils.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "onnx/onnx_pb.h"
+#include "core/graph/graph.h"
+#include "core/graph/indexed_sub_graph.h"
+#include "core/graph/function.h"
+#include "core/graph/schema_registry.h"
+
+namespace onnxruntime {
+namespace function_utils {
+
+std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const Graph& graph,
+                                                       const IndexedSubGraph& nodes_to_fuse);
```

Comment:
Would you please add a comment to say when to use the first CreateSchema function and when to use the second?

Reply:
sure.